extensions/spaces/libspaces.m

```diff
@@ -131,26 +131,28 @@ static int spaces_windowsForSpace(lua_State *L) { // NOTE: wrapped in init.lua
     return 1 ;
 }
 
-/// hs.spaces.moveWindowToSpace(window, spaceID) -> true | nil, error
+/// hs.spaces.moveWindowToSpace(window, spaceID, force) -> true | nil, error
 /// Function
 /// Moves the window with the specified windowID to the space specified by spaceID.
 ///
 /// Parameters:
 ///  * `window`  - an integer specifying the ID of the window, or an `hs.window` object
 ///  * `spaceID` - an integer specifying the ID of the space
+///  * `force` - a boolean specifying whether the window should be tried to move even if the spaces aren't compatible
```

Comment:
This should really say `an optional boolean`

Reply:
Fixed.